doc/api/crypto.md

```diff
@@ -590,7 +590,7 @@ added: v0.7.1
 -->
 
 * `autoPadding` {boolean} **Default:** `true`
-* Returns: {Cipher} for method chaining.
+* Returns: {Cipher} A Cipher for method chaining.
```

Comment:
Same as https://github.com/nodejs/node/pull/51592#discussion_r1470049773 for all the changes in this file